client/app/scripts/superdesk-publish/publish.js

```diff
@@ -205,8 +229,21 @@
             newItem.output_channel_id = item.output_channel_id;
             newItem.unique_name = item.unique_name;
             newItem.destination = item.destination;
+            return newItem;
+        };
+
+        $scope.scheduleToSend = function(item) {
+            var queueItems;
+
+            if (angular.isDefined(item)) {
+                queueItems = $scope.buildNewSchedule(item);
```

Comment:
Here maybe we don't need to build a new object. We can pass the same item after setting its _id=null.